Create a uniform style for the action buttons across the application,…

Samgalgo opened the pull request:
… needs more work

## Description
Creates a uniform style for the action buttons in a few views, but soon in all the views for our models.

## Background Information
We want to prevent non-uniform style across our application.

## Changelog

Internal changes
- Refines style of app's buttons in one file stylesheets>application.scss

External changes
- The action buttons are blue (soon to add one or two more colors)and when you hover over them their color changes a little bit.

## Implications for `dev` branch
No implications

## Screenshots/Recordings

(Please include before/after screenshots of any user-facing changes)

## PR Checklist

- [ ] Write clear, easy to read description of PR
- [ ] Remove all text from PR description that is in (parenthesis)
- [ ] Add and label screenshots (if dealing with UI. if not, delete this line)
- [ ] Remove debugging statements like `console.log`, `byebug`, `puts`, `prints`, `p`, `pp`

## How does this PR make you feel?

Happy and more confident!

Samgalgo committed:
Create a uniform style for the action buttons across the application, needs more work

myang1010 reviewed (changes requested):
Thanks for the work. Just a few changes needed. Thanks.

Samgalgo committed:
Styles all the action buttons in the app

Samgalgo commented on app/assets/stylesheets/application.scss:
```diff
@@ -63,7 +64,22 @@ h1 {
+    } 
 }
 
+.btn-space {
```
Noted ;) thank you for your advice, Ming. 
I think I decided to delete this part before I got to see the review, but what you mention makes total sense.

Samgalgo commented on app/assets/stylesheets/scaffolds.scss:
```diff
@@ -12,7 +12,7 @@ pre {
   padding: 10px;
   font-size: 11px; }
-
+/*
```
I thought I would need this code again in the future that is why I commented it out and left it in there but also I did not leave any comments specifying why I left it in there, oopsie. In case I want to keep some code by commenting it out, would adding a comment explaining its purpose would be ok? Thank you!

Samgalgo commented on app/assets/stylesheets/scaffolds.scss:
```diff
@@ -21,7 +21,9 @@ a:visited {
 a:hover {
   color: #fff;
-  background-color: #000; }
+  background-color: none; }
```
Hi Ming, I removed this part because these lines were preventing the links to look how I wanted them to look. These lines made the background dark and the links didn't look aesthetical. I believe these lines are the original code that came in the app when doing scaffold, I remember having the same issue in my personal final project and not being able to find where this style was coming from. I am really happy I was able to find how to get rid of this style as It was driving me crazy in my personal project.

Samgalgo commented on app/assets/stylesheets/application.scss:
```diff
@@ -63,7 +64,22 @@ h1 {
-    color: black;
+
+
+.btn-primar {
```
Yeea I had the feeling that this would be an issue because the name sounds incomplete. Thank you for mentioning that defining my own `btn-primary` is ok,  I thought I would have issues with bootstrap's already defined class.

Samgalgo committed:
Fix changes requested on review

Samgalgo commented:
Hi @myang1010 Ming, I tried to resolve the ### conflicts which i belive is just this extra symbol **{**, I believe I changed everything on my local repo and pushed it. Afterward I notice the  **This branch has conflicts that must be resolved
Use the [web editor]
to resolve conflicts.
Conflicting files
app/assets/stylesheets/application.scss**

I clicked on resolve conflicts and when i deleted the extra **{** it gave me the following msgs **plz see attached photo** i am unsure if I have to do something. Thank you!!!

(https://github.com/CTDPracticum/unite/pull/11/conflicts) or the 
![Screen Shot 2022-09-12 at 1 51 05 PM](https://user-images.githubusercontent.com/82852120/189722503-82750419-ad49-4de5-9980-1408f559f164.png)

Samgalgo committed:
Fixed conflict

myang1010 commented on app/views/groups/index.html.erb:
```diff
@@ -17,17 +17,17 @@
-    <%= link_to 'New Group', new_group_path, class: "btn btn-primary" %>
-  </div>
-</div>
+   
```
[Code Hygiene]
Always check what you've committed. These empty lines could be removed next time.
